doc/asciidoc/docker/data/docker-compose.yml

```diff
@@ -0,0 +1,41 @@
+version: '3'
+services:
+  neo4j:
+    image: neo4j:3.4
+    hostname: neo4j
+    container_name: neo4j
+    ports:
+    - "7474:7474"
+    - "7687:7687"
+    depends_on:
+    - kafka
+    volumes:
+    - $HOME/neo4j/3.4/plugins:/plugins
```

Comment:
I guess this is something that this then is externalized. Perhaps we can put a comment into the YML.